src/blocks/services/service/edit.js

```diff
@@ -217,19 +235,33 @@ class Edit extends Component {
 			],
 		];
 
-		if ( attributes.showCta ) {
+		if ( showCta ) {
 			TEMPLATE.push( [ 'core/button', {} ] );
 		}
 
 		return (
 			<Fragment>
+				<BlockControls>
+					{ imageUrl && (
+						<ImageURLInputUI
+							url={ href || '' }
+							onChangeUrl={ this.onSetHref }
+							linkDestination={ linkDestination }
+							mediaUrl={ imageUrl }
+							mediaLink={ imageUrl }
```

Comment:
@jrtashjian I need to pass the right value here so we can link the image to the actual page.

This is what's done on the core/image block : 

```
mediaUrl={ ( image && image.source_url ) || url }
mediaLink={ image && image.link }
```